code/game/objects/items/pneumaticCannon.dm

```diff
@@ -315,7 +315,7 @@
 	pressureSetting = 2
 	range_multiplier = 3
 	throw_amount = 1
-	maxWeightClass = 4 //a single magspear or spear
+	maxWeightClass = 20 //a single magspear or spear
```

Comment:
```suggestion
	maxWeightClass = WEIGHT_CLASS_BULKY
```

This number doesn't need to be changed from 4, but since you pointed it out anyway it should be set to the proper weight class define instead of hardcoded as a 4.

Reply:
Fixed in this commit